### ISSUE TITLE
Use ES6 syntax to import

### ISSUE THREAD

**Issue by mukarramali** [schmiedeone/vacabot#7]
Right now the way we import files and modules in our code follows ES5 syntax.
We have the opportunity to refactor it and move to ES6.
You can look it up here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import

**Comment by opensourceally** [comment]
Can I pick this up?

**Comment by Bhavesh27** [comment]
@mukarramali Can I Pick this up seems @opensourceally is occupied?

**Comment by opensourceally** [comment]
I have completed this today. But in the spirit of the assignment take it @Bhavesh27 and complete it.

**Comment by Bhavesh27** [comment]
@mukarramali Added ES6 syntax support. https://github.com/schmiedeone/vacabot/pull/23 Some major changes. Please do tell if any thing breaking. I tried covering most out of it.

**Comment by PPInfy** [comment]
@Bhavesh27 - The same can be done by mjs in node.js though it is experimental. Did you try the same? There might not be any need of Babel as it node 13.x here.

**Comment by Bhavesh27** [comment]
@PPInfy using .mjs extension is experimental + we have to declare project type as module. Babel is smarter choice here it converts the ES7/ES6 syntax to ES5.

**Comment by PPInfy** [comment]
https://blog.logrocket.com/you-dont-need-babel-with-node/

**Comment by Bhavesh27** [comment]
> https://blog.logrocket.com/you-dont-need-babel-with-node/

Yeah i referenced this only. It clearly depicts we have to enable experimental mode for .mjs file and declare project type as module in package.json. In the last section it mentions use cases where we might need the babel transpiler
